.github/CODEOWNERS

```diff
@@ -274,7 +274,7 @@
 /sdk/communication/azure-communication-chat/           @ankitarorabit @minnieliu @Azure/azure-sdk-communication-code-reviewers
 
 # PRLabel: %Communication - Email
-/sdk/communication/azure-communication-email/          @yogeshmo
+/sdk/communication/azure-communication-email/          @kagbakpem
```

Comment:
Can we also add this `@Azure/azure-sdk-communication-code-reviewers` as codeowners as this library also belongs to ACS.

Reply:
I would suggest azure-sdk-write-communication as https://github.com/orgs/Azure/teams/azure-sdk-communication-code-reviewers only has one user that isn't even on the team any more.